testonly/hammer/hammer.go

```diff
@@ -409,7 +443,7 @@ type writeWorker struct {
 
 func newWriteWorker(s *hammerState) *writeWorker {
 	return &writeWorker{
-		mapWorker: newWorker(s.cfg, rand.New(s.cfg.RandSource)),
+		mapWorker: newWorker(s.cfg, s.cfg.EPBias, rand.New(s.cfg.RandSource)),
```

Comment:
Does this bias not need filtering out read-only stuff?

Reply:
It will eventually. This PR is a refactoring that enables these functional changes. There are still underlying broken things about the local copies of the data structures that need resolving before we can change this though.